skema/rest/workflows.py

```diff
@@ -66,6 +68,43 @@ async def equations_to_amr(data: schema.EquationImagesToAMR):
     return res.json()
 
 
+# equation images -> mml -> latex
+@router.post("/images/equations-to-latex", summary="Equations (images) → MML → LaTeX")
+async def equations_to_latex(data: UploadFile):
+    """
+    Converts images of equations to LaTeX.
+
+    ### Python example
+    ```
+    Endpoint for generating LaTeX from an input image.
+
+    ### Python example
+    ```
+    import requests
```

Comment:
@ualiangzhang , did you test to see if the include Python example here works with your new endpoint?

Reply:
Yes. I tested it before. It worked for me.